### ISSUE TITLE
Table comments breaking target

### ISSUE THREAD

**Issue by vitoravancini** [datamill-co/target-redshift#45]
Hello everyone, I've been using target-postgres for almost an year and I'm now beggining to use target-redshift. Thank you for all the awesome work.

I'm having an issue when the target code tries to read the table comment metadata. I'm not sure why this is happening, and I was able to make it work changing two lines of code, maybe showing which lines I've changed and my json schema you could point me to something I'm getting wrong.

` 
comment = cur.fetchone()[0]

if comment:
    try:
        comment_meta = json.loads(comment)
    except:
        self.LOGGER.exception('Could not load table comment metadata')
        raise
else:
    comment_meta = None`

to 

` 
comment = cur.fetchone()
        
if comment and len(comment) > 0:
    comment = comment[0]
    try:
        comment_meta = json.loads(comment)
    except:
        self.LOGGER.exception('Could not load table comment metadata')
        raise
else:
    comment_meta = None
`

and my json schema is

`{
    "streams": [{
        "stream_alias": "batidas",
        "stream": "batidas",
        "tap_stream_id": "batidas",
        "schema": {
            "selected": true,
            "type": ["null", "object"],
            "properties": {
                "NSR": {
                    "type": ["null", "string"]
                },
                "NREP": {
                    "type": ["null", "string"]
                },
                "PIS": {
                    "type": ["null", "string"]
                },
                "Data": {
                    "type": ["null", "string"]
                },
                "Hora": {
                    "type": ["null", "string"]
                },
                "Latitude": {
                    "type": ["null", "string"]
                },
                "Longitude": {
                    "type": ["null", "string"]
                }
            }
        },
        "metadata": [{
            "metadata": {
                "selected": true,
                "table-key-properties": ["NSR", "NREP", "PIS"]
            },
            "breadcrumb": []
        }]
    }]
}`

Thank you in advance!

**Comment by vitoravancini** [comment]
just now realized that this file is from target-postgres, not target-redshift.

**Comment by AlexanderMann** [comment]
@vitoravancini yup, that's expected. They are very related, built on the same codebase.

Is this a long lived table? ie, has it been syncing data for some time and this just now popped up? What kind of replication are you using? We haven't been seeing any issues with this code on our end, but just to be sure we could easily rerun the tests herein.

**Comment by jo-carney** [comment]
@vitoravancini @AlexanderMann Were you able to resolve this issue? My team is currently dealing with a similar problem
